plugin/src/main/groovy/gorm/tools/databinding/EntityMapBinder.groovy

```diff
@@ -84,4 +113,45 @@ class EntityMapBinder implements MapBinder {
         conversionHelpers[converter.targetType] << converter
     }
 
+    @SuppressWarnings(["EmptyCatchBlock", "BitwiseOperatorInConditional", "CatchException"])
+    private List getBindingIncludeList(final Object object) {
+        List includeList = Collections.emptyList()
+        try {
+            Class<? extends Object> objectClass = object.getClass()
+            if (CLASS_TO_BINDING_INCLUDE_LIST.containsKey(objectClass)) {
+                includeList = CLASS_TO_BINDING_INCLUDE_LIST.get(objectClass)
+            } else {
+                Field whiteListField = objectClass.getDeclaredField(DefaultASTDatabindingHelper.DEFAULT_DATABINDING_WHITELIST)
+                if (whiteListField != null) {
+                    if ((whiteListField.getModifiers() & Modifier.STATIC) != 0) {
+                        Object whiteListValue = whiteListField.get(objectClass)
+                        if (whiteListValue instanceof List) {
+                            includeList = (List)whiteListValue
+                        }
+                    }
+                }
+                if (!Environment.getCurrent().isReloadEnabled()) {
+                    CLASS_TO_BINDING_INCLUDE_LIST.put(objectClass, includeList)
+                }
+            }
+        } catch (Exception e) {
+        }
+        return includeList
+    }
+
+    protected preprocessValue(propertyValue) {
+        if(propertyValue instanceof CharSequence) {
+            String stringValue = propertyValue.toString()
```

Comment:
toString or cast?

Reply:
Just used from the grails data binder.. can just check for string and cast